staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -336,41 +327,6 @@ func (d *DiscoveryClient) ServerVersion() (*version.Info, error) {
 	return &info, nil
 }
 
-// SwaggerSchema retrieves and parses the swagger API schema the server supports.
-// TODO: Replace usages with Open API.  Tracked in https://github.com/kubernetes/kubernetes/issues/44589
-func (d *DiscoveryClient) SwaggerSchema(version schema.GroupVersion) (*swagger.ApiDeclaration, error) {
```

Comment:
We don't know if there is any client-go user depends on swagger 1.2. Could you send an email to kubernetes-dev to ask if anyone objects remove swagger 1.2 from client-go? We can proceed if no one objects.

Reply:
Thanks for looking at it! Yeah, doing it right now.

Reply:
No news? Shall we proceed?

Reply:
Let's proceed.